### ISSUE TITLE
A11y_ML .NET Model Builder_Image Classification-Data Preview_ScreenReader: Narrator/NVDA is not announcing number of items present in a folder in data preview section.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1629]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev3; #Visual Studio Client; #WCAG1.3.1; #A11yPreview; #Narrator; #NVDA; #Screen Reader;

### Environment Details:
Microsoft Visual Studio

### Re-pro Steps:

1. Turn on Narrator/NVDA. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#) ex: WpfApp
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Image classification" and press enter
5. Select "Local (CPU)" environment and tab till "Next" button and press enter
6. In Data tab, Tab till "Browse" button and select the input file
7. After uploading data, in data preview section tab till folders and verify whether Narrator.NVDA is reading no of items present in a folder.

### Actual Result:
Narrator/NVDA is not announcing number of items present in a folder in data preview section.

### Expected Result:
Narrator/NVDA should announce number of items present in a folder in data preview section.

### User Impact:
Screen reader users will not know how many items are present in a folder if Narrator is not announcing the data.

**Comment by beccamc** [comment]
Link to a build available here https://devdiv.visualstudio.com/_apis/resources/Containers/8663755/VSSetup_Release17?itemPath=VSSetup_Release17%2FMicrosoft.ML.ModelBuilder.Gallery.Dev17.vsix

**Comment by beccamc** [comment]
Here is a link the most recent Model Builder build. I’ve tested it on VS 17.0.0 Preview 5 

Model Builder: https://devdiv.visualstudio.com/_apis/resources/Containers/8681620/VSSetup_Release17?itemPath=VSSetup_Release17%2FMicrosoft.ML.ModelBuilder.Gallery.Dev17.vsix
VS: https://aka.ms/vs/17/pre/vs_Enterprise.exe

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. NVDA/Narrator is reading number of images present in each tab clearly. Hence closing this bug.

![1629_Narrator_fixed](https://user-images.githubusercontent.com/83570638/133381317-731b244d-88d0-48d7-b4f9-44c752affb15.png)
![1629_NVDA_fixed](https://user-images.githubusercontent.com/83570638/133381320-104f3a18-b146-4d28-93f6-6d70ad57d8e3.png)
